### DISCUSSION TITLE
Query frequency to avoid AUT_00021 "Request limit has been reached"

### DISCUSSION THREAD

**Discussion by philipsoeberg** [persandstrom/python-verisure#175] [Q&A]
I have for years used a 15s poll frequency by my system where command `vsure -c vsure.cookie <user> <pass> --smart-lock` is run. Result then parsed to update state of alarm system, enabling me to on/off my alarm system using doorman v2n combined with verisure vbox-micro and their smartlock modules.

Recently my vsure command has returned
`verisure.session.LoginError: {"errorGroup": "FORBIDDEN","errorCode": "AUT_00021","errorMessage": "Request limit has been reached"}`

I have tried to increase frequency to 90s (which is unusable for my system, but just for testing) but the AUT_00021 remain (less frequent, but does occur).

I cannot read anything on verisures website and contacting them they never respond back. Does the community have any idea about what the acceptable frequency is for polling verisures servers?

Is there another approach available, like event-streaming from verisures servers?

**Comment by persandstrom** [comment]
Hi,

If I remember correctly verisue has requested people to keep the requsts down to 1 every 30 minutes. This was years ago, so I'm not sure if the information is accurate any more. However, I believe the default request interval is 60 seconds for Homeassistan and that seems to be accepted, I do not know why you get the error when polling every 90s, but maybe you have to just not poll for a day or so.

I think you can configure so that a mail is sent when the lockstatus changes and monitor you mailbox instead. I have not tried this myself.
